pkg/agent/controller/networkpolicy/fqdn_test.go

```diff
@@ -744,3 +747,81 @@ func TestOnDNSResponse(t *testing.T) {
 		})
 	}
 }
+func TestFQDNCacheMinTTL(t *testing.T) {
+	currentTime := time.Now()
+	testIPv4 := "192.168.1.1"
+	testIPv6 := "2001:db8::1"
+	testFQDN := "fqdn-test-pod.lfx.test"
+	getDNSMsg := func(ttl uint32) *dns.Msg {
+		return &dns.Msg{
+			Question: []dns.Question{
+				{Name: testFQDN, Qtype: dns.TypeA, Qclass: dns.ClassINET},
+				{Name: testFQDN, Qtype: dns.TypeAAAA, Qclass: dns.ClassINET},
+			},
+			Answer: []dns.RR{
+				&dns.A{
+					Hdr: dns.RR_Header{
+						Name:   testFQDN,
+						Rrtype: dns.TypeA,
+						Class:  dns.ClassINET,
+						Ttl:    ttl,
+					},
+					A: net.ParseIP(testIPv4),
+				},
+				&dns.AAAA{
+					Hdr: dns.RR_Header{
+						Name:   testFQDN,
+						Rrtype: dns.TypeAAAA,
+						Class:  dns.ClassINET,
+						Ttl:    ttl,
+					},
+					AAAA: net.ParseIP(testIPv6),
+				},
+			},
+		}
+	}
+	tests := []struct {
+		name            string
+		expectedTTL     time.Time
+		fqdnCacheMinTTL uint32
+		ttl             uint32
+	}{
+		{
+			name:            "Response TTL less than FQDNCacheTTL",
+			expectedTTL:     currentTime.Add(10 * time.Second),
```

Comment:
to improve readability of the test case definitions, it would be nice if `expectedTTL` followed the same "format" as `fqdnCacheMinTTL` and `ttl`

I would suggest using a plain integer for `expectedTTL` (like for the other 2 fields), and making the necessary conversion in the test implementation (`t.Run`)